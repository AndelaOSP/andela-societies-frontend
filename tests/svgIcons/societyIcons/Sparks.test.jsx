import React from 'react';
import { shallow } from 'enzyme';

import SparksIcon from '../../../src/components/svgIcons/societyIcons/Sparks';

describe('<SparksIcon />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<SparksIcon />);
    expect(wrapper).toMatchSnapshot();
  });
});
