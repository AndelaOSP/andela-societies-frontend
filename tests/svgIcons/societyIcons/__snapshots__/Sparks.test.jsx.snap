// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SparksIcon /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <style>
            .cls-1{fill:#fff;}
          </style>
        </defs>,
        <title>
          sparks-white-icon
        </title>,
        <polygon
          className="cls-1"
          points="52.42 0 52.42 32.92 61.42 45.42 27.92 45.42 41.5 34.83 41.5 18.92 25 18.92 52.42 0"
        />,
        <polygon
          className="cls-1"
          points="9 45.42 9 12.5 0 0 33.5 0 19.92 10.58 19.92 26.5 36.42 26.5 9 45.42"
        />,
      ],
      "data-name": "Layer 1",
      "id": "Layer_1",
      "viewBox": "0 0 61.42 45.42",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <style>
            .cls-1{fill:#fff;}
          </style>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": ".cls-1{fill:#fff;}",
          },
          "ref": null,
          "rendered": ".cls-1{fill:#fff;}",
          "type": "style",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "sparks-white-icon",
        },
        "ref": null,
        "rendered": "sparks-white-icon",
        "type": "title",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "points": "52.42 0 52.42 32.92 61.42 45.42 27.92 45.42 41.5 34.83 41.5 18.92 25 18.92 52.42 0",
        },
        "ref": null,
        "rendered": null,
        "type": "polygon",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "points": "9 45.42 9 12.5 0 0 33.5 0 19.92 10.58 19.92 26.5 36.42 26.5 9 45.42",
        },
        "ref": null,
        "rendered": null,
        "type": "polygon",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <style>
              .cls-1{fill:#fff;}
            </style>
          </defs>,
          <title>
            sparks-white-icon
          </title>,
          <polygon
            className="cls-1"
            points="52.42 0 52.42 32.92 61.42 45.42 27.92 45.42 41.5 34.83 41.5 18.92 25 18.92 52.42 0"
          />,
          <polygon
            className="cls-1"
            points="9 45.42 9 12.5 0 0 33.5 0 19.92 10.58 19.92 26.5 36.42 26.5 9 45.42"
          />,
        ],
        "data-name": "Layer 1",
        "id": "Layer_1",
        "viewBox": "0 0 61.42 45.42",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <style>
              .cls-1{fill:#fff;}
            </style>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": ".cls-1{fill:#fff;}",
            },
            "ref": null,
            "rendered": ".cls-1{fill:#fff;}",
            "type": "style",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "sparks-white-icon",
          },
          "ref": null,
          "rendered": "sparks-white-icon",
          "type": "title",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "points": "52.42 0 52.42 32.92 61.42 45.42 27.92 45.42 41.5 34.83 41.5 18.92 25 18.92 52.42 0",
          },
          "ref": null,
          "rendered": null,
          "type": "polygon",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "points": "9 45.42 9 12.5 0 0 33.5 0 19.92 10.58 19.92 26.5 36.42 26.5 9 45.42",
          },
          "ref": null,
          "rendered": null,
          "type": "polygon",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
